### ISSUE TITLE
Question: --printer PaperSize usage

### ISSUE THREAD

**Issue by gulien** [unoconv/unoconv#485]
Hello 👋 

Any idea how the option `--printer PaperSize=YxZ` works?

Currently it does nothing, but maybe it should be used with `--printer PaperFormat=A_FORMAT`?

(related: https://github.com/thecodingmachine/gotenberg/issues/36)

**Comment by hanfengcan** [comment]
I have the same Question, maybe PaperSize is inoperative

**Comment by regebro** [comment]
Yeah, it seems to have stopped working, together with export options (`-e`).

**Comment by regebro** [comment]
I've run out of ideas here, I don't know why this doesn't work.  I've noticed that in my printer settings it tells me I can't change the PaperFormat, but that I can change the PrinterSize, but that also doesn't work. I've been trying to find examples of how this should be done, but can't find any.

**Comment by gulien** [comment]
With unoconv, `PaperFormat` works in my case (using formats like `A3` or `A4`).

The "printer" is your physical printer? Or some kind of LibreOffice service used for PDF conversion?

**Comment by regebro** [comment]
It's my physical printer in my case.

**Comment by DuJiming** [comment]
I also have this problem.
